docs/mine/lotus/manage-storage-deals.md

```diff
@@ -1,33 +1,139 @@
 breadcrumb: 'Manage storage deals'
 ---
 
 # {{ $frontmatter.title }}
 
 {{ $frontmatter.description }}
 
-::: callout
-This page is a _stub_! We are working on expanding these docs. If you have ideas, please contribute to improving this page.
-:::
+During the lifetime of a miner, Filecoin network clients, will query the storage price-ask advertised by the miner and initiate deals. Deals go through several phases which, at high level are:
 
-## Exploring deals
+1. Data transfer (for online deals) or data import (for offline deals)
+2. Sealing sector with deal data (miner)
+3. Proving (every 24 hours)
 
-While the Lotus Miner is running, the `lotus-miner` application can be used to manage and configure some aspects of the deals like the storage-ask details. The following command shows the self-documented commands around storage deals:
+The following sections provides insights into the different ways Lotus can be used to manage several parts of the storage-deal process.
+
+## Enabling and disabling deals
+
+There are two ways to enable and disable new storage deals in the miner:
+
+- Editing the `[DealMaking]` options in the [miner configuration file](miner-configuration.md) and [restarting the miner](miner-lifecycle.md). Since re-starting the miner is a delicate operation, it is better to let Lotus handle it by:
+- Using the `lotus-miner storage-deals selection` commands.
+
+For example, to disable storage deals:
+
+```sh
+lotus-miner storage-deals selection reject --online
+lotus-miner storage-deals selection reject --offline
+```
+
+The commands above will automatically update the values in the `config.toml` file.
+
+You can verify the current status with:
+
+```sh
+lotus-miner storage-deals selection list
+```
+
+You can re-enable storage deals with:
+
+```sh
+lotus-miner storage-deals selection reset
+# Verify that they have been enabled
+lotus-miner storage-deals selection list
+```
+
+Note that the values above affect to new deals. Ongoing deals will still have to be honored.
+
+## Setting the ask price
+
+One of the most important aspects of accepting new deals will be the conditions and price that miners request. Incoming deals are evaluated on these conditions and automatically accepted or rejected by Lotus miners.
+
+Storage prices (and other conditions) are set with the `lotus-miner storage-deals set-ask` command. For example:
+
+```sh
+lotus-miner storage-deals set-ask \
+  --price 100000000000 \
+  --verified-price  100000000000 \
+  --min-piece-size 56KiB \
+  --max-piece-size 32GB
+```
+
+The above command sets the price for deals to _100000000000 attoFIL_ (100 nanoFIL) per Gibibyte per Epoch. That means that a client will have to pay 100 nanoFIL every 30 seconds for each Gibibyte stored. If the client wants 5GiB stored over the course of a week, the total price will be: \_5GiB \* 100nanoFIL/GiB_Epoch \* 20160 Epochs = 10080 microFIL`.
```

Comment:
```suggestion
The above command sets the price for deals to `100000000000 attoFIL` (`100 nanoFIL`) per GiB, per epoch. This means, a client will have to pay `100 nanoFIL` every 30 seconds for each GiB stored. If the client wants 5GiB stored over the course of a week, the total price will be: `5GiB * 100nanoFIL/GiB_Epoch * 20160 Epochs = 10080 microFIL`.
```

Reply:
I'm confused about this equation. Should it be: `(5GiB * ( 100nanoFIL / GiB_epoch ) * 20160 Epochs = 10080 microFIL` ?

Reply:
There's no difference, but can write as `5GiB * 20160 epochs * 100nanoFIL / GiB_epoch`